packages/gatsby-theme-docs/src/components/card.js

```diff
@@ -47,61 +48,83 @@ WrapWith.propTypes = {
   children: PropTypes.node.isRequired,
 };
 
+const testValue = `Certainly! Here's an example of a simple JavaScript code snippet with four lines:
+
+\`\`\`javascript
+const name = 'John';
+console.log('Hello, ' + name + '!');
+
+const age = 25;
+console.log('You are ' + age + ' years old.');
+\`\`\`
+
+This code declares a variable \`name\` with the value "John" and logs a greeting message to the console. It also declares a variable \`age\` with the value 25 and logs a message about the age.
+`;
+
+const TestMultiMarkdown = () => (
+  <div>
+    {markdownFragmentToReact(testValue, { pre: MultiCodeBlockMarkdownWrapper })}
```

Comment:
FYI I used the cards component to test the use case...once approved, I will revert all the changes in this file.